org.palladiosimulator.analyzer.slingshot.behavior.systemsimulation/src/org/palladiosimulator/analyzer/slingshot/behavior/systemsimulation/SeffSimulationBehavior.java

```diff
@@ -135,26 +134,26 @@ private UserRequestFinished finishUserRequest(final SEFFInterpretationContext en
 	 * @return
 	 */
 	private SEFFInterpretationProgressed continueInParent(final SEFFInterpretationContext entity) {
-
-		final SeffBehaviorWrapper seffBehaviorHolder = entity.getBehaviorContext().getParent().get();
-
-		final SEFFInterpretationContext seffInterpretationContext = SEFFInterpretationContext.builder()
-				.withAssemblyContext(entity.getAssemblyContext())
-				.withBehaviorContext(seffBehaviorHolder.getContext())
-				.withCaller(entity.getCaller())
-				.withRequestProcessingContext(entity.getRequestProcessingContext())
-				.build();
+		final SEFFInterpretationContext seffInterpretationContext = entity.getParent()
+				.orElseThrow(() -> new IllegalStateException("Every child context must have a parent"));
```

Comment:
This is causing problems for me. 

I'm running an extended version of the minimal example. i have a loadbalancer component, i.e. a SEFF with an external call in a nested behaviors (branch). the example runs just fine on the `master` branch,  but with this additional parent fiel, i end up with an `IllegalStateException` here. 

```  
[Worker-5: Launching minimal fcfs closed] INFO : Even dispatched at 13.5: SEFFInterpretationFinished(SEFFInterpretationFinished-2B5858C9)
[Worker-5: Launching minimal fcfs closed] INFO : return to parent
[Worker-5: Launching minimal fcfs closed] ERROR: An error occurred: IllegalStateException:: Every child context must have a parent
java.lang.IllegalStateException: Every child context must have a parent
	at org.palladiosimulator.analyzer.slingshot.behavior.systemsimulation.SeffSimulationBehavior.lambda$0(SeffSimulationBehavior.java:138)
	at java.base/java.util.Optional.orElseThrow(Optional.java:403)
	at org.palladiosimulator.analyzer.slingshot.behavior.systemsimulation.SeffSimulationBehavior.continueInParent(SeffSimulationBehavior.java:138)
	[...]
```

When i check the `SeffInterpreter` the new field `parent`of the context is indeed not set. 
As im not yet quite sure about the meaning of the parent field, i don't know how to fix this. 

I suggest we discuss this in the meeting this afternoon? ó_ò